catalog/ui/search-ui/standard/src/test/js/wd/shared.js

```diff
@@ -9,6 +9,61 @@ var chaiAsPromised = require('chai-as-promised');
 var wd = require('wd');
 var fs = require('fs');
 var path = require('path');
+var url = argv.url || 'http://localhost:8888/';
+var newline = '\n';
+var indentation = '';
+var stackTrace = [];
+var browser;
+var failureScreenshotPath = path.join(__dirname, '../../../..', 'target/webapp/images/failures');
+var screenshotPath = path.join(__dirname, '../../../..', 'target/webapp/images');
+/*
+ Called when a test fails.  It constructs a screenshot name by removing problematic characters from
+ the full test name.  It then prints out the stack trace of commands and saves a screenshot to the
+ failure directory.
+ */
+function handleError(testName){
+    var screenshotName = testName.split(' ').join('-').split('/').join('-').split(':').join('-')+'.png';
```

Comment:
Not a deal-breaker, but maybe something like `.replace(/( |\/|:)/g, '-')` would be more concise.